fast64-animations_updater/fast64-animations_updater_status.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I'm not a huge fan of ``-`` in filenames, I'd go with ``fast64_animation_updater`` instead, idk what other people think about this though

Reply:
CRYING

Reply:
oohhhh that was an accidental commit lol